node/src/main/scala/coop/rchain/node/storage/LmdbManager.scala

```diff
@@ -0,0 +1,130 @@
+package coop.rchain.node.storage
+
+import java.nio.ByteBuffer
+import java.nio.file.{Files, Path}
+
+import cats.effect.concurrent.{Deferred, Ref}
+import cats.effect.{Concurrent, Sync}
+import cats.syntax.all._
+import coop.rchain.shared.Log
+import coop.rchain.store.{KeyValueStore, KeyValueStoreManager}
+import enumeratum.{Enum, EnumEntry}
+import org.lmdbjava.{DbiFlags, Env, EnvFlags}
+
+object LmdbManager {
```

Comment:
Might be rename this to `LmdbStoreManager` to match `InMemoryStoreManager` and `KeyValueStoreManager `?

Reply:
Also would be nice to have a comment explaining what's this like in InMemoryStoreManager

Reply:
@nzpr I renamed to `LmdbStoreManager`. I'm not sure what you mean in the second comment. Does my https://github.com/rchain/rchain/pull/2935#issuecomment-636499108 for `KeyValueStoreManager` answers to this?

Reply:
@tgrospic I was just for adding a short comment like `InMemoryStoreManager` has, explaining what is this for. But nvm, I believe the name should be self explaining.

Reply:
I've added a comment as part of reporting cache PR.
https://github.com/rchain/rchain/blob/391b8f228/casper/src/main/scala/coop/rchain/casper/storage/LmdbStoreManager.scala#L19-L26